rmw_connext_shared_cpp/src/topic_names_and_types.cpp

```diff
@@ -71,7 +71,7 @@ get_topic_names_and_types(
   node_info->subscriber_listener->fill_topic_names_and_types(no_demangle, topics);
 
   // Copy data to results handle
-  if (topics.size() > 0) {
+  if (!topics.empty() > 0) {
```

Comment:
Incomplete change - mixing book and integer comparison.

Reply:
Fixed